doc/e2ee.md

```diff
@@ -69,6 +69,22 @@ This allows the decoder to understand the frame a bit more and makes it decode t
 video. This also means the SFU does not know (ideally) that the content is end-to-end encrypted and there are no
 changes in the SFU required at all.
 
+If the signature bit is set on the frame trailer, there is an additional fixed-length signature that is located
+between the counter and the trailing bit. The signature is generated as
+  Signature = Sign(signatureKey, Authentication Tag)
+and covers the current frame. Not every frame is signed but there will be periodic
+signatures on all SSRCs and streams. This prevents the impersonation attacks described in
+  https://tools.ietf.org/html/draft-omara-sframe-00#section-4.4
+We currently sign every frame, despite the overhead this incurs (132 bytes per frame currently).
```

Comment:
Where does the (magical) 132 number come from? Is it the length of the P-521 signature?

Reply:
explained a bit below so removed here